operations-stdlib/src/jvmTest/kotlin/encoding/EncodeTest.kt

```diff
@@ -0,0 +1,58 @@
+package encoding
+
+import TestInput
+import io.kotest.core.spec.style.FunSpec
+import io.kotest.datatest.withData
+import io.kotest.matchers.shouldBe
+
+class EncodeTest: FunSpec({
```

Comment:
Please add polish characters test too. And Test with already encoded partially string. We have cases where you get URL with additional paramenter that is already encoded.